spec/yaml/swiftnav/sbp/signing.yaml

```diff
@@ -154,12 +154,16 @@ definitions:
             fill: u8
             size: 4
             desc: The last 4 bytes of the certificate's SHA-1 fingerprint
+        - n_signature_bytes:
+            type: u8
+            desc: >
+              Number of bytes to use of the signature field.
         - signature:
             type: array
             fill: u8
-            size: 64
+            size: 73
```

Comment:
72 is the maximum from the research I did, but we can leave it at 73 if you'd like to leave some wiggle room -- see https://snav.slack.com/archives/C04RCE6GWRH/p1678904497727939